cmake/KokkosConfig.cmake.in

```diff
@@ -46,12 +46,12 @@ include(FindPackageHandleStandardArgs)
 #   and target architectures and generates a fatal error if it was not.
 #
 #   kokkos_check(
-#     [DEVICES <devices>...] # Set of backends (e.g. "OpenMP" and/or "Cuda")
-#     [ARCH <archs>...]      # Target architectures (e.g. "Power9" and/or "Volta70")
-#     [OPTIONS <options>...] # Optional settings (e.g. "PROFILING")
-#     [TPLS <tpls>...]       # Third party libraries
-#     [RESULT <result>]      # Set a variable that indicates the result of the
-#                            # check instead of a fatal error
+#     [DEVICES <devices>...]   # Set of backends (e.g. "OpenMP" and/or "Cuda")
+#     [ARCH <archs>...]        # Target architectures (e.g. "Power9" and/or "Volta70")
+#     [OPTIONS <options>...]   # Optional settings (e.g. "PROFILING")
+#     [TPLS <tpls>...]         # Third party libraries
+#     [RETURN_VALUE <result>]  # Set a variable that indicates the result of the
```

Comment:
@jjwilke Why did you go with `RETURN_VALUE`?  Can you point to another function that use that naming convention?  I was trying to think of CMake functions and came across [`execute_process()`](https://cmake.org/cmake/help/latest/command/execute_process.html) that uses `RESULTS_VARIABLE`.

Reply:
`exec_program` and `ctest_test` e.g.  The moral of the story is probably that CMake itself doesn't follow a consistent name.  Either is fine. Sorry, I didn't mean to bike shed.  I just wanted to be a bit clearer than `RESULT`.